OctoAwesome/OctoAwesome.Client/OctoGame.cs

```diff
@@ -84,8 +76,23 @@ public OctoGame() : base()
 
             Assets = new AssetComponent(this);
             Components.Add(Assets);
```

Comment:
Hier kommt es zur NullReferenceException

Reply:
Das ist ein Property aus dem Basis Klasse. Was habt ihr da bitte geändert?

Reply:
Die Property wird local in diesem Stand ausgeblendet also nein das ist nicht die Prop aus der Basisklasse ;)